odoo/models.py

```diff
@@ -2832,7 +2850,7 @@ def _field_to_sql(self, alias: str, fname: str, query: (Query | None) = None, fl
         result to make method :meth:`~odoo.api.Environment.execute_query` flush
         the field before executing the query.
         """
-        property_name = None
+        property_name = ''
```

Comment:
Property should not be there, but handled somewhere else. Here we manage simple fields.
Except for translations. (for now?)